graphdatascience/system/system_endpoints.py

```diff
@@ -39,10 +39,6 @@ def is_licensed(self) -> bool:
     def debug(self) -> DebugProcRunner:
         return DebugProcRunner(self._query_runner, f"{self._namespace}.debug", self._server_version)
 
-    @property
-    def config(self) -> DebugProcRunner:
-        return DebugProcRunner(self._query_runner, f"{self._namespace}.config", self._server_version)
-
 
 class IndirectSystemEndpoints(CallerBase):
```

Comment:
Good question. It's because `IndirectSystemEndpoints` (and the other endpoint classes) is never instantiated explicitly, but rather just inherited by other classes that themselves inherit eg. `UncallableNamespace` if appropriate for that class. The endpoint classes are mixins one could say